inlineplz/main.py

```diff
@@ -31,6 +31,8 @@ def main():
     parser.add_argument('--zero-exit', action='store_true')
     parser.add_argument('--install', action='store_true')
     parser.add_argument('--max-comments', default=25, type=int, help='maximum comments to write')
+    parser.add_argument('--prefix', type=str)
+    parser.add_argument('--delete-outdated', action='store_true')
```

Comment:
since this is destructive behavior, might be better to have it default to false/off